src/rdkafka.h

```diff
@@ -2781,13 +2781,15 @@ void rd_kafka_queue_cb_event_enable (rd_kafka_queue_t *rkqu,
  */
 
 
-#define RD_KAFKA_OFFSET_BEGINNING -2  /**< Start consuming from beginning of
-				       *   kafka partition queue: oldest msg */
-#define RD_KAFKA_OFFSET_END       -1  /**< Start consuming from end of kafka
-				       *   partition queue: next msg */
-#define RD_KAFKA_OFFSET_STORED -1000  /**< Start consuming from offset retrieved
-				       *   from offset store */
-#define RD_KAFKA_OFFSET_INVALID -1001 /**< Invalid offset */
+#define RD_KAFKA_OFFSET_LAST_STABLE -3 /**< Start consuming from last stable
```

Comment:
this is here because the [design doc](https://docs.google.com/document/d/11Jqy_GjUGtdXJK94XGsEIK7CP1SnQGdp2eF0wSw9ra8/edit#) says (for `ListOffsets`):

> The schema is exactly the same as version 1, but we now support a new sentinel timestamp in the request (-3) to retrieve the LSO.

However, now I look, I can't see any trace of this in the broker code. Removing this altogether. Also removing the change I made to `rd_kafka_offset_reset` that uses this constant. can you check that this looks right, or if it's not clear tell me to investigate further.